index.html

```diff
@@ -63,100 +63,25 @@ <h2>My Recent Works</h2>
-
-
-                    <a href="https://github.com/yashodhicy/Microverseday3pr"><button class="see_project">See
-                            Project</button></a>
-                </div>
-            </div>
-            <div class="work-card">
-                <div class="text">
-                    <h3>Multi-Post Stories <br>Gain+Glory</h3>
-                    <ul>
-                        <li class="tags"><button> Ruby on Rails</button></li>
-                        <li class="tags"><button> CSS</button></li>
-                        <li class="tags"><button> JavaScript</button></li>
-                        <li class="tags"><button> HTML</button></li>
-                    </ul>
-
-
-                    <a href="https://github.com/yashodhicy/Microverseday3pr"><button class="see_project">See
-                            Project</button></a>
-                </div>
-            </div>
-            <div class="work-card">
-                <div class="text">
-                    <h3>Multi-Post Stories <br>Gain+Glory</h3>
-                    <ul>
-                        <li class="tags"><button> Ruby on Rails</button></li>
-                        <li class="tags"><button> CSS</button></li>
-                        <li class="tags"><button> JavaScript</button></li>
-                        <li class="tags"><button> HTML</button></li>
-                    </ul>
-
-
-                    <a href="https://github.com/yashodhicy/Microverseday3pr"><button class="see_project">See
-                            Project</button></a>
-                </div>
+        <!-- popup menu -->
+        <!-- <div id="popup-overlay"></div> -->
+        <div id="popup" class="hide">
+            <button id="cancel-btn2"></button>
+            
+            <img id="popup-image">
+            <div id="content">
+            <div id="primary-text">
+            <h2 id="popup-title">test</h2>
+            <ul id="popup-techs"></ul>
             </div>
-            <div class="work-card">
-                <div class="text">
-                    <h3>Multi-Post Stories <br>Gain+Glory</h3>
-                    <ul>
-                        <li class="tags"><button> Ruby on Rails</button></li>
-                        <li class="tags"><button> CSS</button></li>
-                        <li class="tags"><button> JavaScript</button></li>
-                        <li class="tags"><button> HTML</button></li>
-                    </ul>
-
-
-                    <a href="https://github.com/yashodhicy/Microverseday3pr"><button class="see_project">See
-                            Project</button></a>
-                </div>
+            <p id="popup-description"></p>
+            <div class="button-group">
+                <button id="popup-livelink" class="button linkbutton">See Live<img class="livelink" src="images/seelive.png"></button>
+                <button id="popup-source" class="button linkbutton">See Source<img class="livelink" src="images/gitwhite.png"></button>
```

Comment:
- [ ] Great work fixing the spacing between the `see live` & `see source` buttons. You however dint reduce the spacing between the links. Please see below and compare the figma against your implementation
**Figma**
![link1](https://user-images.githubusercontent.com/99656687/231671733-bfe20fd9-4b2d-44dc-b07b-f510773959ea.png)


**Your design**
![links](https://user-images.githubusercontent.com/99656687/231671455-c5dfada5-bd33-4b25-bfbc-48ffbb5db6bb.png)